server/proxy/proxy.go

```diff
@@ -86,6 +88,45 @@ func (p *Proxy) Notify(cc *apps.Context, subj apps.Subject) error {
 	return nil
 }
 
+func (p *Proxy) NotifyRemoteWebhook(app *apps.App, data []byte, webhookPath string) error {
+	if !app.GrantedPermissions.Contains(apps.PermissionRemoteWebhooks) {
+		return utils.NewForbiddenError("%s does not have permission %s", app.AppID, apps.PermissionRemoteWebhooks)
+	}
+
+	up, err := p.upstreamForApp(app)
+	if err != nil {
+		return err
+	}
+
+	var datav interface{}
+	err = json.Unmarshal(data, &datav)
+	if err != nil {
+		// if the data can not be decoded as JSON, send it "as is", as a string.
+		datav = string(data)
+	}
+
+	// TODO: do we need to customize the Expand & State for the webhook Call?
+	creq := &apps.CallRequest{
+		Call: apps.Call{
+			Path: path.Join(apps.PathWebhook, webhookPath),
+		},
+		Context: p.conf.GetConfig().SetContextDefaultsForApp(app.AppID, &apps.Context{
+			ActingUserID: app.BotUserID,
+		}),
+		Values: map[string]interface{}{
+			"data": datav,
+			"path": webhookPath,
```

Comment:
Do we really need to put the `path` into `Values`? `Call.Path` does contain it anyway. 1/5 that we should not duplicate information in a `Call`?